docs/diagrams/src/PostFreesurferPipeline-long.GeneralScheme.puml

```diff
@@ -0,0 +1,41 @@
+@startuml PrePostFreesurferPipeline-long general scheme
+|p| Longitudinal TP
+|t| Template
+
+|p|
+-[hidden]->
+group Timepoint processing Stage 1
+:Run FreeSurfer2CaretConvertAndRegisterNonlinear;
+#palegreen: MNI152/Native/MSMSulc
+/{L|R}.sphere_rot.surf.gii;
+note
+generate average 
+surfaces, then stop
+end note
+stop
+end group
+|t|
+group Template processing
+:Run FreeSurfer2CaretConvertAndRegisterNonlinear
+until MSMSulc registration code;
+:Average MNI152/Native/MSMSulc
+/{L|R}.sphere_rot_average.surf.gii
```

Comment:
Are we averaging files named `sphere_rot_average`?  It seems like we would be averaging `sphere_rot`, with `sphere_rot_average` as the output (as shown in the figure).

Reply:
We're averaging sphere.rot.native.surf. I updated this .puml file to reflect that